src/test/resources/pages/studentHomeTypicalHTML.html

```diff
@@ -93,7 +95,7 @@ <h1>
           <td>
             Graced Feedback Session
           </td>
-          <td>
+          <td data-date-stamp="${datetime.now.iso8601utc}">
```

Comment:
The build will only pass between 10am and 4pm UTC due to #8640; don't worry about the build failure

Reply:
I've merged a quick fix so subsequent builds should pass.